src/transformers/models/audio_spectrogram_transformer/configuration_audio_spectrogram_transformer.py

```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+# Copyright 2022 Google AI and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" AudioSpectrogramTransformer model configuration"""
+
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+AUDIO_SPECTROGRAM_TRANSFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    # TODO update to appropriate organization
+    "nielsr/audio-spectrogram-transformer-finetuned-audioset-10-10-0.4593": "https://huggingface.co/nielsr/audio-spectrogram-transformer-finetuned-audioset-10-10-0.4593/resolve/main/config.json",
```

Comment:
The checkpoint name is starting to be a bit long, would it make sense to abbreviate audio-spectogram-transformer to ast?

Reply:
Ok I'll update to `ast`!